index.html

```diff
@@ -80,7 +80,7 @@ <h2>Stages</h2>
     </td>
     <td>None
     <td>Major
-    <td>Polyfills / demos
+   <td>Demos and experimental reference implementations (implementations that modify global built-in objects should not be published as libraries or used in production-deployed code; implementations that do not modify global built-in objects may be published)
```

Comment:
Moving this to a new conversation: I also think the “Experimental” in Stage 2 should be further clarified to:

> Experimental engine implementations and reference implementations (implementations that modify global objects should still not be published as libraries or used in production-deployed code)

I think redundancy for the sake of clarity is desirable in this case. “Experimental” alone no longer makes sense, since Stage 1 now also uses the word “experimental”.